### PR TITLE
Use become to avoid root login in AWS

### PR DESCRIPTION
**Pull request by rsevilla87**
We should avoid login as root and enabling SSH login with this user due to the  security risks it creates.

Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### CODE REVIEW

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Review by mohit-sheth — APPROVED** [review]
LGTM

File: OCP-4.X/roles/install-on-aws/tasks/main.yml [hunk 1]
```diff
@@ -172,27 +172,21 @@
   when: openshift_install_release_image_override == "" and openshift_install_binary_url == ""
 
 - name: ensure that .kube dir exists
-  become: "{{item.become}}"
   file:
     path: "{{item.path}}"
     state: directory
   with_items:
     - path: /root/.kube
```
Comment (dry923) [inline OCP-4.X/roles/install-on-aws/tasks/main.yml:179]: if we are not root and the become is removed how are we going to access /root/.kube ?

Comment (rsevilla87) [reply OCP-4.X/roles/install-on-aws/tasks/main.yml:179]: Become is already set at playbook level, so it affect all tasks

Comment (dry923) [reply OCP-4.X/roles/install-on-aws/tasks/main.yml:179]: Ah didn't see that. In that case why didc you go through add add it to the install-on-aws|azure|gcp files?

Comment (chaitanyaenr) [reply OCP-4.X/roles/install-on-aws/tasks/main.yml:181]: @rsevilla87 we might want to apply the changes in install-on-aws playbook to gcp, azure and osp.

**Pull request merged**